Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -911,6 +919,10 @@ void readDatalink(void){
             case SET_KIVALUES:
                 setKValues(GAIN_KI,(float*)(&cmd->data));
                 break;
+            case SET_GAINS:
+                int type = (int*) (&cmd->data);
```

Comment:
Yeah it doesn't look right. Has this code been tested? Look at the NEW_WAYPOINT case if you want to see how multi part commands (commands with multiple values) are interpreted